awx/api/views.py

```diff
@@ -1285,7 +1285,7 @@ class ProjectUpdatesList(SubListAPIView):
     new_in_13 = True
 
 
-class ProjectUpdateView(RetrieveAPIView):
+class ProjectUpdateView(RetrieveAPIView, GenericAPIView):
```

Comment:
I thought that `RetrieveAPIView` inherited from genericAPIview, so this shouldn't be needed

Reply:
Indeed it does - updating..
http://www.django-rest-framework.org/api-guide/generic-views/#retrieveapiview